moto/databrew/__init__.py

```diff
@@ -0,0 +1,4 @@
+from .models import databrew_backend
+
+databrew_backends = {"global": databrew_backend}
+mock_databrew = databrew_backend.decorator
```

Comment:
Using the `global`-region in the backends is only used in global services such as S3, and as far as I can tell `databrew` is regional. 
See how the CloudTrail-service is setup as an example: https://github.com/spulec/moto/blob/master/moto/cloudtrail/__init__.py

Reply:
Ah great. will push a fix for that shortly.